index.js

```diff
@@ -0,0 +1,104 @@
+'use strict'
+
+const Hp = require('hemera-plugin')
+const AWS = require('aws-sdk')
+const DynamoStore = require('./store')
+const StorePattern = require('hemera-store/pattern')
+const Joi = require('joi')
+
+function hemeraDynamoStore(hemera, opts, done) {
+  let topic = 'dynamo-store'
+
+  var dynamo = new AWS.DynamoDB(opts.dynamodb)
+  var db = new AWS.DynamoDB.DocumentClient(opts.dynamodb, dynamo)
+
+  function createDb() {
+    return dynamo
+  }
+
+  hemera.decorate('dynamoStore', {
+    createDb
+  })
+
+  hemera.add(StorePattern.create(topic), function (req, cb) {
+    const store = new DynamoStore(db)
+    store.create(req, cb)
+  })
+
+  hemera.add({
+    topic,
+    cmd: 'removeById',
+    id: Joi.required(),
+    collection: Joi.string().required()
+  }, function (req, cb) {
+    const store = new DynamoStore(db)
+    store.removeById(req, cb)
+  })
+
+  hemera.add({
+    topic,
+    cmd: 'updateById',
+    id: Joi.required(),
+    collection: Joi.string().required(),
+    UpdateExpression: Joi.string().required,
+    ConditionExpression: Joi.string(),
+    ExpressionAttributeNames: Joi.object(),
+    ExpressionAttributeValues: Joi.object()
+  }, function (req, cb) {
+    const store = new DynamoStore(db)
+    store.updateById(req, cb)
+  })
+
+  hemera.add({
+    topic,
+    cmd: 'findById',
+    id: Joi.required(),
+    collection: Joi.string().required(),
+    ProjectionExpression: Joi.string(),
+    ExpressionAttributeNames: Joi.object()
+  }, function (req, cb) {
+    const store = new DynamoStore(db)
+    store.findById(req, cb)
```

Comment:
If you prefer you can also use promise style. It's much better than callback style in this case. E.g https://github.com/hemerajs/hemera-mongo-store/blob/master/index.js#L97